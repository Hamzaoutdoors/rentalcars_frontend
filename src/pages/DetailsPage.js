import CarDetails from '../components/cars/CarDetails';

const DetailsPage = () => (
  <CarDetails />
);

export default DetailsPage;
