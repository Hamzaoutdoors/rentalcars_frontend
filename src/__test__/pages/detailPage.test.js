import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import DetailsPage from '../../pages/DetailsPage';
import store from '../../redux/configureStore';

it('render the car detail page correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <Router>
          <DetailsPage />
        </Router>
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
