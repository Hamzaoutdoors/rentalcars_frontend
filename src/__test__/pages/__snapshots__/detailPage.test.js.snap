// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render the car detail page correctly 1`] = `
<div
  className="sc-jrQzAO gURVnT"
/>
`;
